test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverClientTestBase.cs

```diff
@@ -124,7 +124,7 @@ public abstract class SenderReceiverClientTestBase
             TestUtility.Log("Sleeping 10 seconds...");
             await Task.Delay(TimeSpan.FromSeconds(10));
 
-            DateTime lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
+            var lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
```

Comment:
@nemakam Agree. With `lockedUntilUtcTime` I would argue you're dealing with a date time that cares for UTC and not an offset. If you need to emphasize the offset, you'll do so with a variable name. For example: `localTimeWithOffset` or `localTimeWithTimezone` would indicate that you're looking at a time with offset component. 

Now, for the argument
> But in this case unless you open this in a IDE, you cant be sure whether the object returned is DateTime / DateTimeOffset or any other custom implementation.

The codebase is no longer a few files. To actually read it and understand, a review of a diff on GH is not enough. It either requires to pull the code down and see it all, or requires to be familiar with the code. I'd say if you pull the code down, you will be using an IDE.

We need to understand how vehicles operate to drive those, but do we really need to know to the low level how things work exactly? You know there's an engine, not Engive v6 3.5L blah blah blah. I see `var` as engine. Specifications (if needed) can be a part of the variable name.

Hopefully I have managed to convey my POV 😃